NetInventors/sw.ext.neti_tags
# #28762 - Delete protecting

Testing 1

1. Go to the plugin configuration at the backend
2. Set deletprotecting on true
3. Create a tag and add him some relations
4. Delete this tag, you now see the message that you can't delete this tag

Testing 2
1. Go to the plugin configuration at the backend
2. Set deletprotecting on false
3. Create a tag and add him some relations
4. Delete this tag, you now see the modified delete message
